sdk/search/azure-search-documents/azure/search/documents/_index/_generated/operations/_documents_operations.py

```diff
@@ -53,7 +53,7 @@ def count(
         :raises: ~azure.core.exceptions.HttpResponseError
         """
         cls = kwargs.pop('cls', None)  # type: ClsType[int]
-        error_map = kwargs.pop('error_map', {})
+        error_map = kwargs.pop('error_map', {404: ResourceNotFoundError, 409: ResourceExistsError})
```

Comment:
If possible, would it make more sense to have a method return these so you can just change the method in one place later and it return a map instead?

Reply:
This is in generated code

Reply:
This is in the generated code